src/builder.rs

```diff
@@ -624,6 +629,48 @@ impl CpModelBuilder {
         }
     }
 
+    /// Add all solution hints from a solution.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use cp_sat::builder::{CpModelBuilder, LinearExpr};
+    /// # use cp_sat::proto::CpSolverStatus;
+    /// let mut model = CpModelBuilder::default();
+    /// let x = model.new_int_var([(50, 100)]);
+    /// let y = model.new_int_var([(0, 51)]);
+    /// model.add_hint(x, 75);
+    /// model.add_hint(y, 42);
+    /// model.add_lt(x, y);
+    /// let response = model.solve();
+    /// assert_eq!(response.status(), CpSolverStatus::Optimal);
+    /// assert!(x.solution_value(&response) < y.solution_value(&response));
+    /// assert_eq!(50, x.solution_value(&response));
+    /// assert_eq!(51, y.solution_value(&response));
+    /// model.del_hints();
+    /// model.add_hints(response.solution);
+    /// model.del_all_cst();
+    /// let response = model.solve();
+    /// assert_eq!(response.status(), CpSolverStatus::Optimal);
+    ///  
+    /// ```
+    pub fn add_hints(&mut self, solution: Vec<i64>) {
+        let hints = self
+            .proto
+            .solution_hint
+            .get_or_insert_with(Default::default);
+
+        for (i, _) in solution.iter().enumerate().take(self.proto.variables.len()) {
```

Comment:
And this function will not work, as if you have done `add_hints`  before, they will not be erased.

Reply:
Ok